### DISCUSSION TITLE
how can we 'close the render window' in readthedocs or jupyter

### DISCUSSION THREAD

**Discussion by panziyi1** [pyvista/pyvista#3386] [Q&A]
In my program, some resource management activities run after close the render window on left click the 'x' button (bash).

These activities seens not running when the render window is converted and displayed in jupyter cell. 

**Does some code  can do the same job in jupyter as 'close the render window' in bash?**

**Comment by banesullivan** [comment]
Calling `plotter.close()` should do it. See https://docs.pyvista.org/api/plotting/_autosummary/pyvista.Plotter.close.html?highlight=close#pyvista.Plotter.close
